client/src/scenes/BootScene.ts

```diff
@@ -0,0 +1,53 @@
+import { Scene } from "phaser";
+import { HathoraClient } from "@hathora/client-sdk";
+import { RoomConnection } from "../connection";
+
+const client = new HathoraClient(process.env.APP_ID as string, process.env.COORDINATOR_HOST);
+
+class BootScene extends Scene {
+  constructor() {
+    super("scene-boot");
+  }
+
+  preload() {
+    this.load.image("player", "player.png");
+    this.load.image("bullet", "bullet.png");
+  }
+
+  create() {
+    getToken().then((token) => {
+      getRoomId(token).then((roomId) => {
+        const connection = new RoomConnection(client, token, roomId);
+        connection.connect();
+        const hathoraUser = HathoraClient.getUserFromToken(token);
+
+        this.scene.start('scene-game', {
+          connection,
+          hathoraUser
```

Comment:
instead of passing the user I'd prefer to just pass the token here

Reply:
Okay, I'll refactor 👍